packages/smooth_app/lib/l10n/app_en.arb

```diff
@@ -282,10 +282,25 @@
         "description": "Indicator inside the darkmode switch (system default)"
     },
     "thanks_for_contributing": "Thanks for contributing!",
-    "@contributors": {
+    "contributors_label": "Mobile app contributors",
```

Comment:
```suggestion
    "contributors_label": "They are building the app",
```
Perhaps something emotional like that ?

Reply:
That's up to you ;)
That will be the entry in the menu

Reply:
let's try that, and we could turn the "Contribute" app into "Help improve the App"